src/Microsoft.AspNetCore.Blazor.Server/BlazorConfig.cs

```diff
@@ -29,6 +29,9 @@ private BlazorConfig(string assemblyPath)
             var configLines = File.ReadLines(configFilePath).ToList();
             SourceMSBuildPath = configLines[0];
 
+            if (string.IsNullOrEmpty(SourceMSBuildPath) || SourceMSBuildPath == ".")
+                SourceMSBuildPath = assemblyPath;
```

Comment:
Could you put braces on separate lines around this one? The code style here is always to write:

```
if (condition)
{
    action();
}
```

... and not just

```
if (condition)
    action();
```

Reply:
Thanks for the information, i will do it like this in futur contribution on the project.